### PR TITLE
readme for LMA deployment tool

### PR DESCRIPTION
**Pull request by emmanuel-jr**
Adding documentation to help users understand how to use our Local Multiplayer Agent deployment tool
This includes the requirements and basic steps needed to run the deployment script

### CODE REVIEW

File: LocalMultiplayerAgent/DeploymentTool/readme.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "SessionConfig": {
+        "SessionId": "ba67d671-512a-4e7d-a38c-2329ce181946",
+        "SessionCookie": null,
+        "InitialPlayers": [ "Player1", "Player2" ]
+    }
+    `
+
+## Steps:
+
+- Configure [build settings](./DeploymentSettings.json). Feel free to reconfigure DeploymentSettings.json to suit your needs. Example:
```
Comment (emmayspark) [inline LocalMultiplayerAgent/DeploymentTool/readme.md:25]: We can provide examples of DeploymentSettings.json along with MultiplayerSettings.json for different builds - Linux Container, Windows Container and Windows Process

File: LocalMultiplayerAgent/DeploymentTool/readme.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿After testing their game server(s) on LMA, you may want to go ahead and deploy them.
```
Comment (emmayspark) [inline LocalMultiplayerAgent/DeploymentTool/readme.md:1]: We could add where to deploy them and why we need that

File: LocalMultiplayerAgent/DeploymentTool/readme.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "UseStreamingForAssetDownloads": false,
+    "WindowsCrashDumpConfiguration": {  // WindowsCrashDumpConfiguration is an additional parameter you can add for Windows Container build
+        "IsEnabled": true,
+        "DumpType": 0,
+        "CustomDumpFlags": 6693
+        }
+    `
+
+- Run 
+    - for Windows Process/Container:
```
Comment (emmayspark) [inline LocalMultiplayerAgent/DeploymentTool/readme.md:53]: Can we list out the steps including secret key configuration

File: LocalMultiplayerAgent/DeploymentTool/readme.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+﻿After testing their game server(s) on LMA, you may want to go ahead and deploy them.
+This tool was developed exactly for this purpose; allow you go ahead to deploy your game servers.
```
Comment (emmayspark) [inline LocalMultiplayerAgent/DeploymentTool/readme.md:2]: as I mentioned above, we can add more explanation here - where are the game servers deployed? how do we verify it after running the script? what are the steps?

**Review by emmayspark — COMMENTED** [review]
Great Job !

File: LocalMultiplayerAgent/DeploymentTool/readme.md (hunk 3 above)
Comment (emmanuel-jr) [reply LocalMultiplayerAgent/DeploymentTool/readme.md:53]: Oh shoot, I missed that

File: LocalMultiplayerAgent/readme.md [hunk 5]
```diff
@@ -18,3 +18,4 @@ Once the files are copied over, the LocalMultiplayerAgent file will need to be c
 ## MultiplayerSettings.json
 
 The [MultiplayerSettings.json generator tool](./SettingsJsonGenerator/README.md) can assist you with configuring your MultiplayerSettings.json file, and configuring it for your server.
+The [DeploymentSettings.json](./DeploymentTool/readme.md) can assist you with configuring your DeploymentSettings.json file for running deployment on LMA.
```
Comment (KalonJ-MSFT) [inline LocalMultiplayerAgent/readme.md:21]: recommendation - favor not using acronyms in documentation if you can help it - it helps with searching and understanding if you spell it out.

File: LocalMultiplayerAgent/BuildTool/readme.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+            "ImageDetails": {
+                "Registry": "mcr.microsoft.com",
+                "ImageName": "playfab/multiplayer",
+                "ImageTag": "wsc-10.0.17763.2458",
+                "Username": "",
+                "Password": ""
+             }
+        }
+        ```
+    - [Linux Container on Windows](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/localmultiplayeragent/run-container-gameserver):
```
Comment (emmayspark) [inline LocalMultiplayerAgent/BuildTool/readme.md:39]: nit: I think we can remove "on Windows" here

File: LocalMultiplayerAgent/BuildTool/readme.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+    "BuildId": "",
+    "Region": "",
+    "SessionConfig": {
+        "SessionId": "ba67d671-512a-4e7d-a38c-2329ce181946",
+        "SessionCookie": null,
+        "InitialPlayers": [ "Player1", "Player2" ]
+    }
+    ```
+
+    - Linux Process is currently not supported on Local Multiplayer Agent, so users cannot create a build with this server type
```
Comment (emmayspark) [inline LocalMultiplayerAgent/BuildTool/readme.md:73]: can we add that users cannot create a Linux process build with "this build tool"?  it sounds like we don't support Linux process build

File: LocalMultiplayerAgent/BuildTool/readme.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+        }
+    ]
+}
+```
+
+**Note**: These are other optional parameters for your `BuildSettings.json` you could add, if needed:
+```json5
+"AreAssetsReadonly": false,
+"UseStreamingForAssetDownloads": false,
+"WindowsCrashDumpConfiguration": {  // WindowsCrashDumpConfiguration is an additional parameter you can add for Windows Container build
```
Comment (emmayspark) [inline LocalMultiplayerAgent/BuildTool/readme.md:97]: maybe we can add a link to doc on WindowsCrashDump?

**Review by emmayspark — CHANGES_REQUESTED** [review]

File: LocalMultiplayerAgent/BuildTool/readme.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+    "BuildId": "",
+    "Region": "",
+    "SessionConfig": {
+        "SessionId": "ba67d671-512a-4e7d-a38c-2329ce181946",
+        "SessionCookie": null,
+        "InitialPlayers": [ "Player1", "Player2" ]
+    }
+    ```
+
+    - Since Linux Process is currently not supported on Local Multiplayer Agent, so users cannot create a build with this build tool
```
Comment (yegarci) [inline LocalMultiplayerAgent/BuildTool/readme.md:73]: nit remove "so"

**Review by yegarci — CHANGES_REQUESTED** [review]

**Review by yegarci — APPROVED** [review]

**Review by emmayspark — APPROVED** [review]

**Pull request merged**